### ISSUE TITLE
Add manpages for binaries

### ISSUE THREAD

**Issue by blattms** [OPM/opm-upscaling#333]
```
W: libopm-upscaling-bin: binary-without-manpage usr/bin/cpchop
W: libopm-upscaling-bin: binary-without-manpage usr/bin/cpchop_depthtrend
W: libopm-upscaling-bin: binary-without-manpage usr/bin/cpregularize
W: libopm-upscaling-bin: binary-without-manpage usr/bin/exp_variogram
W: libopm-upscaling-bin: binary-without-manpage usr/bin/grdecldips
W: libopm-upscaling-bin: binary-without-manpage usr/bin/steadystate_test_implicit
W: libopm-upscaling-bin: binary-without-manpage usr/bin/upscale_avg
W: libopm-upscaling-bin: binary-without-manpage usr/bin/upscale_cap
W: libopm-upscaling-bin: binary-without-manpage usr/bin/upscale_cond
W: libopm-upscaling-bin: binary-without-manpage usr/bin/upscale_elasticity
W: libopm-upscaling-bin: binary-without-manpage usr/bin/upscale_perm
W: libopm-upscaling-bin: binary-without-manpage usr/bin/upscale_relperm
W: libopm-upscaling-bin: binary-without-manpage usr/bin/upscale_relperm_benchmark
W: libopm-upscaling-bin: binary-without-manpage usr/bin/upscale_relpermvisc
W: libopm-upscaling-bin: binary-without-manpage usr/bin/upscale_singlephase
W: libopm-upscaling-bin: binary-without-manpage usr/bin/upscale_steadystate_implicit
```

I feel a bit underqualified for this (and might be missing steam). Unfortunately, `help2cmake -N --version-string 2021.04 -h "" <prog>` is not 100% helpful (just as the help printed) and needs quite a bit of editing. Would be cool if someone else would find time for this.

**Comment by joakim-hove** [comment]
Just a crazy idea from one of the footsoldiers: drop packaging of opm-upscaling?

**Comment by alfbr** [comment]
I will do it 🙂, do you have pointer to an existing man-page I can use as template?

**Comment by blattms** [comment]
Look at [compareECL.1](/OPM/opm-common/blob/master/docs/man1/compareECL.1) in opm-common.

You can use `help2cmake -N --version-string 2021.04 -h "" <prog>` to get a rough template, but you will need to edit the output (short desciption should be more than just the program name and meaningful, detailed description, detailed description of parameters.)

**Comment by alfbr** [comment]
Thank you! I will get to it as soon as I can, it might be a few days though.

**Comment by alfbr** [comment]
All necessary man pages are hopefully in place #338

**Comment by akva2** [comment]
closed by #338
